src/index.tsx

```diff
@@ -10,6 +13,13 @@ import './index.css';
 import App from './components/App';
 import * as serviceWorker from './serviceWorker';
 
+FullStory.init({ orgId: 'QNEN8' });
+
+Sentry.init({
+  dsn: 'https://87dd10a700ea41e9a64df50ec9b367c7@sentry.io/2418505',
+  integrations: [ new SentryFullStory('sentry-test') ]
+});
```

Comment:
Would it be a good idea to move these values into a .env file and use React env vars?

See: https://github.com/patrick-fs/fs-react-redux-sentry/blob/master/src/index.js#L12

Reply:
@patrick-fs for the purposes of the webinar, don't' think so. Otherwise you'd just have to copy the contents of my `.env` into yours. The calculus is a bit different if you want other people to clone your repo and run it.